freqtrade/freqai/data_drawer.py

```diff
@@ -645,12 +645,19 @@ def update_historic_data(self, strategy: IStrategy, dk: FreqaiDataKitchen) -> No
                             + 1
                         )
                     except IndexError:
+                        if history_data[pair][tf].iloc[-1]['date'] < df_dp['date'].iloc[0]:
```

Comment:
aside from this PR, i noticed that we use `history_data[pair][tf]` about 5 times in this for-loop alone.

While dict lookups are not "that" expensive - they're also not free - and i think it might improve readability if we can change that to a "single variable", instead of a nested dict lookup.

Reply:
Good point, changed.

Reply:
@robcaulk forgotten to push? or do we aim to change this at a later point?

Reply:
Not only did I forget to push, I somehow forgot to save it as well. Anyways should be in latest commit.